### DISCUSSION TITLE
Just to be safe. A few Questions about Basing my App on kotlin-toolkit

### DISCUSSION THREAD

**Discussion by WreckingBANG** [readium/kotlin-toolkit#524] [Q&A] [answered]
I just wanted to ask if the BSD License and you allow it, that i fork the test-app and build my own Open-Source E-Book Reader with it. I know that the kotlin-toolkit is licensed with the BSD-License but i am not sure if the test-app is also licensed with it.

The second question is, can i release the project under the GPLV3-License?

**Comment by qnga** [comment, marked as answer]
Hi,
Yes, the testapp is licensed under the same BSD-License as the toolkit. I'm not sure I would fork it if I wanted to build a new app, but I guess it's a matter of personal preference.

And yes, you can release your project under the GPL v3. By the way, I would personally support as much as I can such a project, should it grow up.

**Comment by WreckingBANG** [comment]
Thank you for your quick response. The Project is in its early stages. When it is at a point i am happy releasing it, it will be on my GitHub. I would be happy to see contributions from anybody who wants to help.
